src/main/resources/beam-template.conf

```diff
@@ -883,6 +883,7 @@ beam.routing {
     }
     bikeLaneScaleFactor = "double | 1.0"
     bikeLaneLinkIdsFilePath = ""
+    linkRadiusMeters = "double | 9000.0"
```

Comment:
The default seemed to be 10000, any reason for the change here? It is likely the cause for the failing tests.

Reply:
True, it is the reason behind failing tests. The reason we reduced the value is that we assumed that it will work for all cities. In Seattle, we were facing some issues due to 10k value. @nikolayilyin and I agreed to make it 9k.

We have two options:
1. we can update failing test to use 10k value for linkRadiusMeters,
2. make 10k default, and use 9k in seattle base conf

Thoughts?

Reply:
I am a fan of backwards compatibility, so I would go for leaving the default 10k and making the override to 9k in Seattle